src/main.js

```diff
@@ -1,5 +1,6 @@
 import 'babel-polyfill';
 import Vue from 'vue';
+import countdown from 'countdown';
```

Comment:
No need to import this globally, you can just import it in the `<script>` portion of the `.vue` file where you need it.